codespell_lib/data/dictionary.txt

```diff
@@ -26683,6 +26685,7 @@ haveing->having
 havent'->haven't
 havent't->haven't
 havent->haven't
+haveshape->have shape
```

Comment:
I don't know about this one. Missing spaces are a whole new category of typos. We haven't been collecting that kind of typos so far.

Reply:
Agreed, almost didn't include it in the first place.